src/main/tut/index.html

```diff
@@ -547,6 +593,7 @@
 --
 
 … but I'd have to use [shapeless](http://shapeless.io), and this would turn into a different talk entirely.
+<!-- COMMENT -->
```

Comment:
I don't know, _that_ talk has been done ad nauseam, and by much better orators and programers than I - off the top of my head, Travis Brown and Miles Sabin, at least. Thanks for the vote of confidence though :)